utils/audio.py

```diff
@@ -53,6 +55,16 @@ def load_audio_torch(wave_file, fs):
 
     return audio, fs
 
+def save_torch_audio(process_dir, feature_dir, item, wav_torch, fs, overrides=True):
```

Comment:
Can this function be merged with the existing one?
https://github.com/open-mmlab/Amphion/blob/e400c166aca4685de559fde05407994ad3a1f85f/utils/io.py#L49

Or move it to `utils.io.py`.

Reply:
done